paddle/platform/device_context.h

```diff
@@ -74,8 +75,7 @@ class CUDADeviceContext : public DeviceContext {
  public:
   explicit CUDADeviceContext(const GPUPlace gpu_place) : gpu_place_(gpu_place) {
     GPUPlaceGuard guard(gpu_place_);
-    paddle::platform::throw_on_error(cudaStreamCreate(&stream_),
-                                     "cudaStreamCreate failed");
+    PADDLE_ENFORCE(cudaStreamCreate(&stream_), "cudaStreamCreate failed");
```

Comment:
`PADDLE_ENFORCE` means the condition must be `true`. But `CUXXXX_STATUS_SUCCESS` is zero, and false.  So it seems that these lines should be

```cpp
PADDLE_ENFORCE(cudaStreamCreate(&stream_) == CUDA_SUCCESS);
```